notify.js

```diff
@@ -25,7 +25,7 @@
         return typeof item === 'function';
     }
 
-    function Notify(title, options) {
+    function Notify(title, options, params) {
```

Comment:
Could `params` just be an object within `options` maybe? Also is `params` the best naming convention, how about `data`? Thoughts?